app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/FolderController.kt

```diff
@@ -87,21 +87,27 @@ object FolderController {
 
     private fun MutableRealm.deleteOutdatedFolders(remoteFolders: List<Folder>) {
         getFolders(exceptionsFoldersIds = remoteFolders.map { it.id }, realm = this).reversed().forEach { folder ->
```

Comment:
Why do you do a 'reverse'?

Reply:
When we delete in a `RealmList`, it seems the item is instantly removed from the list, and all other items move forward in the list by 1 position.
So when we are going to the next item in the list, we are in fact missing 1 Folder.
But if we go through the list in reverse order, we won't have this issue.